lib/modules/marker.rb

```diff
@@ -0,0 +1,29 @@
+module Codebreaker
+  module Marker
+    def check_win
+      if win?
+        'Congratulation! You win!'
+      elsif @attempts.zero?
+        @hint = 0
+        @game_start = false
+        "Game over! Secret code is #{@secret_code}."
+      else
+        @attempts -= 1
+        '+' * pluses + '-' * (minuses - pluses)
```

Comment:
maybe, it is better to incapsulate multiplication logic inside methods pluses and munuses, if you use them only for generating response strings.
e.x. pluses method will return string with right amount of pluseses and the same with minuses.
now they return some random numbers